### ISSUE TITLE
Request about B-SNN 

### ISSUE THREAD

**Issue by NguyenSakura** [BindsNET/bindsnet#476]
Dear Author,
Could you show your B-SNN code. I tried to re-simulate the B-SNN paper but the accuracy always below 10% event for MNIST dataset. Thanks for the help.

**Comment by Hananel-Hazan** [comment]
What code are you referring to?

**Comment by NguyenSakura** [comment]
Dear Author,
Thanks for quickly reply. I followed your idea in this paper: Exploring the
connection between binary and spiking neural networks. As mention in the
paper. The training BNN without Batchnorm gives the performance of B-SNN is
better. However, I try to run the BNN code (referred by you in Github for
XNOR-Net) without Batchnorm. Unfortunately, the accuracy ( for example for
Cifar10 with VGG) always below 10 %. I confused about BNN
without Batchnorm. If you don't mind can you shared your BNN train and BNN
conversion to B-SNN using VGG-16 for Cifar 100 as mentioned in your
paper. I greatly appreciate it.  Many Thanks


Vào Th 2, 19 thg 4, 2021 vào lúc 01:54 Hananel Hazan <
***@***.***> đã viết:

**Comment by Hananel-Hazan** [comment]
I am sorry, I dont understand what paper are you referring to?

Hananel

**Comment by NguyenSakura** [comment]
I am sorry, I was a mistake with the group "
https://github.com/NeuroCompLab-psu/SNN-Conversion/issues/3. I am sorry
again.

Vào Th 4, 21 thg 4, 2021 vào lúc 08:52 Hananel Hazan <
***@***.***> đã viết:
